src/server/stream_family_test.cc

```diff
@@ -668,13 +668,13 @@ TEST_F(StreamFamilyTest, XTrimInvalidArgs) {
 
   // Include both maxlen and minid.
   resp = Run({"xtrim", "foo", "maxlen", "2", "minid", "1-1"});
-  EXPECT_THAT(resp, ErrArg("MAXLEN and MINID options at the same time are not compatible"));
+  EXPECT_THAT(resp, ErrArg("syntax error"));
```

Comment:
The error that was sent before is compatible with the redis error in this case. I suggest not to change this